### ISSUE TITLE
upload video to server

### ISSUE THREAD

**Issue by ammyGaba** [jamesla/backgroundvideo#35]
Hy 
i want to upload this vedio to server . as stop function return path to video saved . is there any way to get video file .

**Comment by jhayward** [comment]
yes, you can use https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-file/ to read the file and send to the server.
